docs/testing-components/test-driven-development/index.md

```diff
@@ -27,27 +27,25 @@ Imagine we're going to create the `@enact/ui/IconButton` component and that the
 are that it will have a `<Button>` containing an `<Icon>` as its children and that all properties assigned to the IconButton
```

Comment:
Have you considered changing the description to something related to classes, not props?

Reply:
Done